Khan/wonder-blocks
# WB-1069 - Add the Final Set of Props to LabeledTextField

## Summary:
Adding the final set of props to `LabeledTextField`.

This includes:
* placeholder
* light
* style
* testId

Included unit tests to verify these props work as expected (Coverage is 100%).

Issue: https://khanacademy.atlassian.net/browse/WB-1069

## Test plan:
1. Open Styleguidist or React Storybook (See generated links).
2. View the new examples under Form / LabeledTextField.

---

### Light Prop Example
<img width="365" alt="labeled-text-field-light-example" src="https://user-images.githubusercontent.com/60367213/122789039-84c1c080-d27c-11eb-9a6e-30f6aca118a9.png">

### Style Prop Example
<img width="290" alt="labeled-text-field-style-example" src="https://user-images.githubusercontent.com/60367213/122789079-8d19fb80-d27c-11eb-9caa-e82d59b02962.png">
